{{cookiecutter.project_slug}}/static/src/scss/layout/_header.scss

```diff
@@ -1,14 +1,95 @@
 .demo-announcement {
     padding: 0.75rem 0;
-    background: #000000;
-    color: #ffffff;
+    background: $color-primary;
+    color: $color-white;
     text-align: center;
 }
 
-.page-header {
-    padding: $width-gap 0;
+.header .container {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    padding: 1.5rem;
 }
 
-.page-header__buttons {
+.nav--desktop {
```

Comment:
no that's fair! I was coming from the angle of there being barely anything in the header file to start with, but of course that grows with project specific styles. Also theres less temptation to have styles for nav elements in the header file, putting them in two places. Then there's also sometimes navigation in the header and in the footer, with different styles, then I'm not sure if you'd put styles in the header/footer or navigation... 

I want the separation of a `navigation.scss` but the consistency of definitely knowing where the styles will be! What do you think?

Reply:
I'd say let's have a separate `navigation.scss` file and keep it for the primary navigation of the site perhaps? And then all the little bits of `navs` that are in the asides / footers / breadcrumbs and so on can be treated separately and styled either as includes or within their context (e.g. footer). How does that sound?

Reply:
yes that sounds great, let's do that! I'll go with verbose `primary_nav.scss` which i guess encourages a second nav to go in `secondary_nav.scss` ;) happy with namings?

Reply:
ah last question, if nav__toggle is actually in header, and not inside the nav, does that stay in the `header.scss` or go in `primary_nav.scss`?

Reply:
Yes! Awesome! I thought about suggesting an even more descriptive name but stopped myself... dropping `verbose` into a discussion is another one that always gets my vote :)

...oddly I think most people have negative connotations with the word `verbose` in programming, for me it gets immediately translated into "more readable and understandable and generally caring about the next programmer reading this code in a year's time."

So - yes - awesome names!

Reply:
> ah last question, if nav__toggle is actually in header, and not inside the nav, does that stay in the `header.scss` or go in `primary_nav.scss`?

Feels like it could easily go in both - I'd pick `primary_nav.scss` perhaps since it's related to this even if it lives in the header (i.e. it doesn't necessarily relate to the header - it could conceivably be placed elsewhere, but it certainly relates to the navigation). WDYT?

Reply:
same i could go either way, ill go with primary_nav!